src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int ARRAY_INDEX_ZERO = 0;
+    private static final int ARRAY_INDEX_ONE = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
 
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            bufferedWriter.write(createReportData(fromFileName, toFileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data to file " + toFileName, e);
+        }
```

Comment:
![image](https://github.com/mate-academy/jv-work-with-file/assets/20326858/be105e53-ff42-43ae-a75e-0c938d1c52b9)

This is how you should divide program logic. Try to implement these methods.
```suggestion
        List<String> inputLines = readFile(fromFileName);
        String report = createReport(inputLines);
        writeToFile(toFileName, report);
```